### ISSUE TITLE
HttpStress fails to build on Linux, msquic package seems to be outdated

### ISSUE THREAD

**Issue by antonfirsov** [dotnet/runtime#64902]
This line seems to fail now:
https://github.com/dotnet/runtime/blob/179ffff7967f3136d8f8d0d17510a5460dd01adb/src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile#L12

Maybe we shall consider to move the package download logic to a separate `.sh` that only fails for HTTP 3.0 runs?

**Comment by ManickaP** [comment]
Does packages.microsoft.com already have msquic 1.9? If so, we can download from there for now.
But for main, we'll need to consume current msquic --> so we probably should download and just build it.

**Comment by ManickaP** [comment]
BTW, feel free to assign this to me, I'll look into it. 
@antonfirsov also if this is blocking stress CI, comment out the H/3 leg for now and mark this as disabled test. I'll fix it.

**Comment by wfurt** [comment]
https://packages.microsoft.com/debian/11/prod/pool/main/libm/libmsquic/libmsquic_1.9.0_amd64.deb

**Comment by wfurt** [comment]
BTW for that one, we should be able to add repository and use package manager instead of fetching the file directly @ManickaP

**Comment by ManickaP** [comment]
yep for dotnet 6 and msquic 1.9, fetching the lib via package manager should be enough.
for main as-is it's also ok
but once we adjust to msquic 2.0, we'll need to build it ourselves

**Comment by wfurt** [comment]
I agree. But probably we should sort out the main separately - possibly using msquic bits directly if possible.

**Comment by antonfirsov** [comment]
> also if this is blocking stress CI, comment out the H/3 leg for now and mark this as disabled test.

I don't think there is urgency that justifies a PR that does only patching, this has been around for at least a month. Note that the issue breaks the docker build of all HTTP versions on Linux, because the package downloading is implemented in the shared Dockerfile. (Thus the recommendation to move it into a more fault-tolerant script, if possible.)
